Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+    <style>
+        .kapsayici{
+            width: 100%;;
+            height:100vh;
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;   
+          }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         @media all and (max-width: 576px) {
+        .kapsayici{
+            background-color: white;
+        }
+         .anaMenu{
+            flex-direction:column-reverse;
+            height:100%;
+            width: 100%;
+         }
+      }
+         .sagEkran{
+            display: flex;
+            margin-left: 90px;
+            justify-content: center;
+         }
+        .sagEkran>img{
```

Comment:
The image get's stretched on different device sizes. You should avoid it at all costs. Look at this article: https://medium.com/free-code-camp/time-saving-css-techniques-to-create-responsive-images-ebb1e84f90d5 and try to update the code for the image